comprobo_final_project/scripts/simulator/robot.py

```diff
@@ -96,7 +92,7 @@ def step(self, step_freq):
             original_w = self.get_direction()
 
             # From 0 to 2. 1 is neutral
-            noise_factor = 2 * (random.random() * self.noise) + 1
+            noise_factor = 2 * self.noise * (np.random.random() - 0.5)  + 1
```

Comment:
I believe this noise factor (assuming noise of 0.1) comes out to be .9 - 1.1, which makes a lot of sense and sounds good to me. Do consider changing the comment above though, as it was really confusing to me at first, and I had to do the math to figure out what the real intention was.